etc/pyfstat-dev.yml

```
@@ -1,22 +1,31 @@
+# important notes:
+# 1. to ensure correct relative path for the editable mode installation,
+# always create your environment while at the root directory of your pyfstat git clone.
+#
+# 2. to avoid having duplicate lalsuite packages from both conda and pip,
+# please always run as
+# NO_LALSUITE_FROM_PYPI=1 mamba env create -f etc/pyfstat-dev.yml
+
 name: pyfstat-dev
 
 channels:
   - conda-forge
 
 dependencies:
-  - python >=3.8
-  - numpy
-  - matplotlib-base >=2.1
-  - scipy
-  - ptemcee
+  - attrs
   - corner
   - dill
-  - tqdm
+  - lalpulsar >=5.0.0
+  - matplotlib-base >=2.1
+  - numpy
   - peakutils
   - pathos
+  - pip
+  - ptemcee
+  - python >=3.8
   - python-lal >=7.1.5
   - python-lalpulsar >=5.0.0
-  - lalpulsar >=5.0.0
-  - pip
+  - scipy
+  - tqdm
   - pip:
     - -e ../[dev]
```

```suggestion
    - -e .[dev]
```
If you run from the root of PyFstat, should be like this.

---

no it's actually needed like this

```suggestion
    - -e ../[dev]  # the ".." is needed because pip seems to get called from within "etc"
```